### DISCUSSION TITLE
user-guide/plugins/better-notes

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#288] [文档评论区]
# user-guide/plugins/better-notes

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/plugins/better-notes

**Comment by Deborah-Ding** [comment]
怎么用better notes 自动生成各级大纲的思维导呀

**Comment by Jonatic** [comment]
使用better notes输入公式之后再输入中文，有时候会出现无法输入的情况。请问是怎么回事呢，是不是使用方法不对

**Comment by 15haohao** [comment]
想问下better notes设置各层级标题的快捷键是什么

**Reply by LeviHonestus** [reply]
和md一样，几级标题几个#，然后空格

**Reply by gethelp-spec** [reply]
ctrl+shift+对应层级数字

**Comment by YunxiaoZhai** [comment]
使用导出PDF功能时，弹出了windows本地打印窗口（比如让选择类似Microsoft Print to PDF），窗口名称为Printing from Win32 Application，但这个打印界面无法显示要打印的内容，也无法正常打印，请问是bug吗？

**Comment by YangJJ1999** [comment]
添加原文的条目笔记后，会自带引用，但是英文文献的引用为什么还是“等”啊，在word中此引用是正常的“et al”，就是条目笔记中所有英文文献都是“等”

**Reply by BKHANN** [reply]
文献信息里有语言一栏，填en就可以了

**Comment by redmiboo** [comment]
关于无法在标题中插入公式的问题大伙有遇到过吗？就比如输入“# 标题一$\mathbf{\Theta}(t)$”，它不会显示公式

**Reply by MaJiangliang-henu** [reply]
请问怎么显示公式，我$$这样输入并不会显示为公式是需要哪里设置吗

**Reply by BKHANN** [reply]
我这里$$输入完公式后敲一下回车，就会自动显示为公式

**Comment by MaJiangliang-henu** [comment]
在笔记页面，打tex似乎不会显示了，现在有一个f数学公式块，但是这个公式块只能去跨行的公式，不能是行内公式，请问有没有办法行内公式？
